utils/utils.go

```diff
@@ -159,7 +164,7 @@ func CopyMap(src map[string]string) (dst map[string]string) {
 	return
 }
 
-func PrepareLocalPathForUpload(localPath string, useRegExp bool) string {
+func PrepareLocalPathForUpload(localPath string, useRegExp bool, useAnt bool) string {
```

Comment:
I'm not sure it makes sense for this function to receieve two booleans in this case.
It is probably best the function accpets a single argument, which has one of the following 3 values:
1. WildcardPattern
2. RegExp
3. AntPattern
You can define an enum for this. As a reference, look at the ```ItemType``` type we define, to indicate whether we're working with files vs. folders.

The same goes for the ```GetRootPath(pattern, target string, isRegexp, isAnt, preserveSymLink bool) (string, error)``` function signature and a few other functions in the code.